OperationLog.md

```diff
@@ -1,7 +1,8 @@
 - 20230221 - 23399 - awgment-app-account - ACL APIs   
   - Add following roles to existing keycloak under camunda-identity-service client
     - awgment-account-acl-create-or-update      
-    - awgment-account-acl-read        
+    - awgment-account-acl-read   
+  - AUTO INDEX CREATION set as true in application.yaml     
```

Comment:
which application.yaml, where is application.yaml.
Is it already done or needs to be done to make this work correctly?
Please add proper details.